README.md

```diff
@@ -161,15 +161,24 @@ class LokiNoFailureHandler
 }
 
 ```
-Update the config accordingly: 
+Update the config accordingly:
 
 ```php
 'loki' => [
     'driver'    => 'custom',
     'level'     => env('LOG_LEVEL', 'debug'),
     'via'       => \App\Logging\LokiNoFailureHandler::class,
     'formatter_with' => [
-        'labels' => env('LOKI_LABELS', ''),
+        // LOKI_LABELS: app,laravel|env,prod
+        'labels' => env('LOKI_LABELS', '') ? array_reduce(
+            explode('|', env('LOKI_LABELS', '')),
+                function ($carry, $item) {
+                    list($key, $value) = explode(',', $item);
+                    $carry[$key] = $value;
+                    return $carry;
+                },
+                []
+            ) : [],
```

Comment:
Thank you for the contribution.  
But truth be told, although I'm not familiar with Laravel configurations, this sample seems very complicated for something that should be very simple...  

Is there any peculiar reason not to use something simpler like the following options ? 

- parse_str : with a env param value as `LOKI_LABELS: "app=laravel&env=prod"`
```
parse_str(env('LOKI_LABELS', ''), $labels);

[...]

'loki' => [
    [....]
    'formatter_with' => [
        'labels' => $labels,
        [....]
    ],
    [....]
],
```

- json_decode : with a env param value as `LOKI_LABELS: "{app: 'laravel', env: 'prod'}"`
```
[...]

'loki' => [
    [....]
    'formatter_with' => [
        'labels' => json_decode(env('LOKI_LABELS', '[]')),
        [....]
    ],
    [....]
],
```

Reply:
Hey, thanks a lot for the review.
You're absolutely right, it's complicated and I actually don't know anything about PHP; this is my first time with it.

Do you have a preference between the two? I admit I prefer the `parse_str` way.

Reply:
I tested with `parse_str` and shipped it to prod, it's working perfectly fine like that :)
Anyway, the README acts as an example, if one wants to use JSON instead, they're free